CHANGELOG_DEV.md

```diff
@@ -6,6 +6,7 @@ HumHub Changelog
 - Fix #4504: Fix `hasSidebar()` for empty sidebar
 - Fix #4526: `HeaderControlsMenu::init` called twice
 - Fix #4529: Aligned default dropdown text size
+- Fix: Allowing guest like permission
```

Comment:
This seems as we would now allow guests to like content?

Reply:
https://github.com/humhub/humhub/pull/4538

Reply:
No, before this it gave the user an option to give permission to guests which as is, is impossible, I could have worded this p/r better but was in a hurry